medical_physician/data/medical_specialties.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="medical.specialty" id="medical_specialty_gp">
```

Comment:
Oh snap I actually forgot about this. This was part of the [legacy datasets](https://github.com/OCA/vertical-medical/blob/10.0/medical_emr_data/data/health_specialties.xml), I'll bring it in.